src/test/resources/com/puppycrawl/tools/checkstyle/checks/blocks/leftcurly/InputLeftCurlyNlowSwitch.java

```diff
@@ -0,0 +1,53 @@
+package com.puppycrawl.tools.checkstyle.checks.blocks.leftcurly;
+/**
+  * This test-input is intended to be checked using following configuration:
+  *
+  * option = NLOW
+  *
+  */
+public class InputLeftCurlyNlowSwitch {
+
+    public void doStuff() {
+        int x = 1;
+        switch (x) {
+            case 0:
+            { // warn
+                break;
+            }
+            case (1
+                + 0):
+            { // OK, wrapped line
+                break;
+            }
+            case 2: {
+                break;
+            }
+            default
+                :
+            { // OK, wrapped line
```

Comment:
please add case:
```
 case 
    5:
 { // OK, wrapped line
```

Reply:
It is already here https://github.com/checkstyle/checkstyle/pull/6025/files/559ebbb959a8b57e33270f0da8555917fa78b0ce#diff-875b646bca178009b84dda15e5efb282R42

Reply:
```
case 7
+                :
+            { // warn
```

Ok